src/widgets/favoritewidget.cpp

```diff
@@ -50,12 +50,16 @@ QSize FavoriteWidget::sizeHint() const {
 }
 
 void FavoriteWidget::paintEvent(QPaintEvent* e) {
-  QStylePainter p(this);
+  // ignore text descent when vertically centering
+  QFontMetrics fontMetrics = QFontMetrics(this->font());
+  QRect rect = this->rect();
+  rect.setBottom(rect.bottom() - fontMetrics.descent());
```

Comment:
I made a pull request here https://github.com/Ede123/Clementine/pull/1
This way you can avoid call to this->rect() and move everything to constructor...